examples/demo/src/products/GridList.tsx

```diff
@@ -75,6 +83,7 @@ const LoadedGridList = ({ ids, data, basePath, width }) => {
             >
                 {ids.map(id => (
                     <GridListTile
+                        // @ts-ignore
```

Comment:
It seems another attempt was done at migrating this file (see #4592). Would you mind collaborating with the author of that PR instead of including GridList in your PR?

Reply:
sure